### PR TITLE
lightning: add local backend configuration

### PR DESCRIPTION
**Pull request by 3pointer**
### What is changed, added or deleted? (Required)

Add local configuration for lightning new feature.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/tidb-lightning/pull/338

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by glorv** [comment]
https://github.com/pingcap/tidb-lightning/pull/338 this pr  has updated the default value of some field and updated the comments, please adopt these change @3pointer

**Comment by glorv** [comment]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -94,14 +94,22 @@ addr = "172.16.31.10:8287"
 # - ignore：保留已有数据，忽略新数据
 # - error：中止导入并报错
 # on-duplicate = "replace"
+# 当后端是 “local” 时，控制生成 SST 文件的大小，最好跟 tikv 里面的 region 大小保持一致，默认是 96MB。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:97]:
```suggestion
# 当后端是 “local” 时，控制生成 SST 文件的大小，最好跟 TiKV 里面的 Region 大小保持一致，默认是 96 MB。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -94,14 +94,22 @@ addr = "172.16.31.10:8287"
...
+#region-split-size = 100_663_296
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:98]:
```suggestion
# region-split-size = 100_663_296
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -94,14 +94,22 @@ addr = "172.16.31.10:8287"
...
+# 当后端是 “local” 时，一次请求中发送的 KV 数量。
+#send-kv-pairs = 32768
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:100]:
```suggestion
# send-kv-pairs = 32768
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -94,14 +94,22 @@ addr = "172.16.31.10:8287"
...
+# 当后端是 “local” 时，本地进行 KV 排序的路径。如果磁盘性能较低（如使用机械盘），建议设置成与 `data-source-dir` 不同的磁盘中，可有效提升导入性能。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:101]:
```suggestion
# 当后端是 “local” 时，本地进行 KV 排序的路径。如果磁盘性能较低（如使用机械盘），建议设置成与 `data-source-dir` 不同的磁盘，这样可有效提升导入性能。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -94,14 +94,22 @@ addr = "172.16.31.10:8287"
...
+#sorted-kv-dir = ""
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:102]:
```suggestion
# sorted-kv-dir = ""
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 6]
```diff
@@ -94,14 +94,22 @@ addr = "172.16.31.10:8287"
...
+# 当后端是 “local” 时，tikv 写入 kv 数据的并发度，当 lightning 和 tikv 直接网络传输速度超过万兆的时候，可以适当增加这个值。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:103]:
```suggestion
# 当后端是 “local” 时，TiKV 写入 KV 数据的并发度。当 TiDB Lightning 和 TiKV 直接网络传输速度超过万兆的时候，可以适当增加这个值。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 7]
```diff
@@ -94,14 +94,22 @@ addr = "172.16.31.10:8287"
...
+#range-concurrency = 16
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-configuration.md:104]:
```suggestion
# range-concurrency = 16
```

**Pull request merged**